src/olympia/reviewers/forms.py

```diff
@@ -344,7 +344,8 @@ def __init__(self, *args, **kw):
             self.fields['versions'].queryset = (
                 self.helper.addon.versions.distinct().filter(
                     channel=amo.RELEASE_CHANNEL_LISTED,
-                    files__status=amo.STATUS_PUBLIC).order_by('created'))
+                    files__status__in=(amo.STATUS_PUBLIC, amo.STATUS_PENDING)).
```

Comment:
`STATUS_PENDING` is for LWT

Reply:
Uhm, right, but why is the test succeeding anyway?